ydb/docs/en/core/security/audit-log.md

```diff
@@ -1,56 +1,314 @@
+|| [Web login](#web-login) </br>`web-login` | Interactions with the web console authentication widget. | Included in the [basic audit configuration](#enabling-audit-log). ||
+|| [Console](#console) </br>`console` | Database lifecycle operations and dynamic configuration changes. | Included in the [basic audit configuration](#enabling-audit-log). ||
+|#
+
+## Audit event attributes {#audit-event-attributes}
+
+As mentioned, attributes are divided into two groups:
+* Common attributes present in many *audit event sources* and always carry the same meaning.
+* Attributes specific to the source that generates the event.
 
-* File on each {{ ydb-short-name }} cluster node.
-* Agent for delivering [Unified Agent](https://yandex.cloud/docs/monitoring/concepts/data-collection/unified-agent/) metrics.
-* Standard error stream, `stderr`.
+In this section, you will find a reference guide to the attributes in audit events. It covers both common attributes and source-specific ones. For each source, its UID, recorded operations, and configuration requirements are also provided.
 
-You can use any of the listed destinations or their combinations.
+### Common attributes {#common-attributes}
 
-If you forward the stream to a file, access to the audit log is set by file-system rights. Saving the audit log to a file is recommended for production installations.
+The table below lists the common attributes.
 
-Forwarding the audit log to the standard error stream (`stderr`) is recommended for test installations. Further stream processing is determined by the {{ ydb-short-name }} cluster [logging](../devops/observability/logging.md) settings.
+#|
+|| Attribute          | Description ||
+|| `subject`          | Event source SID (`<login>@<subsystem>` format). Unless mandatory authentication is enabled, the attribute will be set to `{none}`. ||
+|| `sanitized_token`  | A partially masked authentication token that was used to execute the request. Can be used to link related events while keeping the original credentials hidden. If authentication was not performed, the value will be `{none}`. ||
+|| `operation`        | Operation name (for example, `ALTER DATABASE`, `CREATE TABLE`). ||
+|| `component`        | Unique identifier (UID) of the *audit event source*. ||
+|| `status`           | Operation completion status.<br/>Acceptable values:<ul><li>`SUCCESS`: The operation completed successfully.</li><li>`ERROR`: The operation failed.</li><li>`IN-PROCESS`: The operation is in progress.</li></ul> ||
+|| `reason`           | Error message. ||
+|| `request_id`       | Unique ID of the request that invoked the operation. You can use the `request_id` to differentiate events related to different operations and link the events together to build a single audit-related operation context. ||
+|| `remote_address`   | IP of the client that delivered the request. ||
+|| `detailed_status`  | The status delivered by a {{ ydb-short-name }} *audit event source*. ||
```

Comment:
The explanation didn't make it clearer. How does it relate to the regular `status` from a few rows above, and what exactly can be present there?